pkg/stanza/operator/parser/container/parser.go

```diff
@@ -272,7 +272,13 @@ func (p *Parser) extractk8sMetaFromFilePath(e *entry.Entry) error {
 		return nil
 	}
 
-	logPath := e.Attributes[logPathField]
+	logPath, ok := e.Attributes[logPathField]
+	if !ok {
+		return fmt.Errorf(
+			"operator '%s' has 'add_metadata_from_filepath' enabled, but 'include_file_path' might be disabled",
+			p.OperatorID)
```

Comment:
Perhaps be more explicit about what is missing and how to fix it?
```suggestion
			"operator '%s' has 'add_metadata_from_filepath' enabled, but the log record attribute '%s' is missing. Perhaps enable the 'include_file_path' option?",
			p.OperatorID,
			logPathField)
```